crates/blockifier/src/execution/sierra_utils.rs

```diff
@@ -128,6 +132,9 @@ pub fn wrap_syscall_handler(syscall_handler: &mut NativeSyscallHandler<'_>) -> S
     SyscallHandlerMeta::new(syscall_handler)
 }
 
+// todo(rodro): String parsing is the lowest way to get from one to the other,
+//              we should find a faster one.
+//              Also, should we implement `From` traits for this?
```

Comment:
As far as I remember we can't implement traits from external crates for structures from external crates due to orphan rules